sparse_strips/vello_hybrid/examples/webgl/src/lib.rs

```diff
@@ -65,21 +65,15 @@ impl RendererWrapper {
         };
 
         let (device, queue) = adapter
-            .request_device(
-                &wgpu::DeviceDescriptor {
-                    label: None,
-                    required_features: wgpu::Features::empty(),
-                    required_limits: wgpu::Limits {
-                        // WGPU's downlevel defaults use a generous number of color attachments
-                        // (8). Some devices (including CI) support only up to 4.
-                        max_color_attachments: 4,
-                        max_texture_dimension_2d: max_texture_size,
-                        ..wgpu::Limits::downlevel_webgl2_defaults()
-                    },
-                    ..Default::default()
+            .request_device(&wgpu::DeviceDescriptor {
+                label: None,
+                required_features: wgpu::Features::empty(),
+                required_limits: wgpu::Limits {
+                    max_texture_dimension_2d: max_texture_size,
```

Comment:
I didn't carefully review this code when it landed the first time. Should this be using the max size from the adapters [`limits`](https://docs.rs/wgpu/latest/wgpu/struct.Adapter.html#method.request_device), instead of getting it manually from the canvas? (cc @taj-p)

Reply:
Yes - I was under the mistaken assumption that we could avoid all the parameter lookups required to construct `Limits`, but I see that [it's created as part of adapter construction](https://github.com/gfx-rs/wgpu/blob/303b0db00ef14431c75929814702aaef07c8cdbb/wgpu-hal/src/gles/adapter.rs#L677) so this is a needless parameter lookup.

I've created a PR for this change [here](https://github.com/linebender/vello/pull/911).